### DISCUSSION TITLE
How to set the completion prompt for pressing < tab > to appear in only one column

### DISCUSSION THREAD

**Discussion by camarayol** [fish-shell/fish-shell#10554] [Q&A] [answered]
![Snipaste_2024-06-07_11-03-00](https://github.com/fish-shell/fish-shell/assets/55070626/eea7dbf6-c4ac-479d-95cf-a4112bd34290)

**Comment by faho** [comment, marked as answer]
Fish does not have an option to force a single-column layout.

In general a multi-column layout is useful because it allows you to select more options faster.
